EIPS/eip-7872.md

```diff
@@ -0,0 +1,44 @@
+---
+eip: 7872
+title: Max blob flag for local builders
+description: Adds a flag to set the maximum number of blobs a local builder will put in a block
+author: Francesco D'Amato <francesco.damato@ethereum.org>, Kevaundray Wedderburn (@kevaundray), Ben Adams (@benaadams), Gajinder Singh (@g11tech), Dustin (@tersec)
```

Comment:
```suggestion
author: Francesco D'Amato <francesco.damato@ethereum.org>, Kevaundray Wedderburn (@kevaundray), Toni Wahrstätter(@nerolation), Alex Stokes (@ralexstokes), Ben Adams (@benaadams), Gajinder Singh (@g11tech), Dustin (@tersec)
```

Reply:
Alex had this idea a while back: https://hackmd.io/ljsmwo6QQ8i3zqToZUPs2A#On-raising-the-blob-count-in-Pectra and Toni did the analysis to show this was safe